homepage/models.py

```diff
@@ -40,15 +41,16 @@ class Req_Code(models.IntegerChoices):
     req_code = models.SmallIntegerField(choices=Req_Code.choices, default=Req_Code.NONE)
 
 
-class User(models.Model):
-    name = models.CharField(max_length=100)
-    email = models.CharField(max_length=50)
-    password = models.CharField(max_length=30)
+class AppUser(models.Model):
+    user = models.OneToOneField(settings.AUTH_USER_MODEL, on_delete=models.CASCADE, primary_key=True)
+
+    def __str__(self):
+        return self.user.username
 
 
 class Review(models.Model):
     course = models.ForeignKey(Course, on_delete=models.CASCADE)
-    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    user = models.ForeignKey(AppUser, on_delete=models.CASCADE)

```

Comment:
Didn't we want to prevent deletion if a user added a review?

Reply:
We agreed that users will not have the ability to delete their account (they will be changed to not-active and will not be able to login anymore if this is something we want and can be easily done)

If we forcefully delete a user, i think it makes sense to also delete all of the user's reviews (over-wise we probably wouldn't forcefully delete his account )

Reply:
Ok, seems I forgot the agreed implementation. 
In that case - LGTM.